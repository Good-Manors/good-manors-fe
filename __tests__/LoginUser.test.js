import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import LoginUser from '../src/containers/LoginUser';

describe('LoginUser', () => {
  it('Renders a LoginUser container', () => {
    const getWrapper = (mockStore = createStore()) => mount(
      <Provider store={mockStore}>
        <LoginUser />
      </Provider>
    );
    expect(getWrapper).toMatchSnapshot();
  });  
});
