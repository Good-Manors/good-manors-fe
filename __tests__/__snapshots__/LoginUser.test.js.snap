// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginUser Renders a LoginUser container 1`] = `[Function]`;
